content/spin/kv-store.md

```diff
@@ -23,7 +23,7 @@ enable_shortcodes = true
 
 ## Key Value Storage With Spin Applications
 
-Spin applications are best suited for event-driven, stateless workloads that have low-latency requirements. Keeping track of the application's state (storing information) is an integral part of any useful product or service. For example, users (and the business) will expect to store and load data/information at all times; during an application’s execution. From [Spin v0.9.0](https://www.fermyon.com/blog/spin-v09), onwards we've added support for key/value storage. This is a storage option for applications that need data in the form of key/value pairs and are satisfied by a BASE consistency model. Workload examples include general value caching, session caching, counters, and serialized application state.
+Spin applications are best suited for event-driven, stateless workloads that have low-latency requirements. Keeping track of the application's state (storing information) is an integral part of any useful product or service. For example, users (and the business) will expect to store and load data/information at all times; during an application’s execution. [Spin](https://www.fermyon.com/blog/spin-v09) has support for applications that need data in the form of key/value pairs and are satisfied by a BASE consistency model. Workload examples include general value caching, session caching, counters, and serialized application state.
```

Comment:
Nit: The 0.9 blog post link is a bit peculiar as the link for the unvarnished "Spin" text.  You could extend the link to "Spin has support."  Or you could just not worry about it, that would be fine too!